go.mod

```diff
@@ -3,16 +3,17 @@ module github.com/marklogic/marklogic-kubernetes
 go 1.17
 
 require (
-	github.com/gruntwork-io/terratest v0.40.6
-	github.com/stretchr/testify v1.7.0
+	github.com/gruntwork-io/terratest v0.41.15
```

Comment:
This go.mod change should not be committed. To make the commit history clear, there should be a designate PR to update the go.mod

Reply:
I used a function that is part of a new version of gruntwork-io/terratest hence go.mod is updated in this PR